src/js/gi/components/AdditionalResources.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+class AdditionalResources extends React.Component {
+
+  render() {
+    return (
+      <span>
```

Comment:
I was using `<span>` just to denote this parent tag was needed by JSX but not in the original project code. Agree, though, and changing this across all components where it was done.